docs/instructions.md

```diff
@@ -165,34 +165,55 @@ ___
 
 # Using Tkinter Designer <small>[[Top](#table-of-contents)]</small>
 
-### Open Tkinter Designer before doing the following steps.
-<br>
+## Required Inputs
+There are some inputs you'll need to collect to be able to use the TKinter Designer.
 
 <a id="using-1"></a>
-## 1. Personal Access Token
+
+### 1. Personal Access Token
 1. Log into your Figma account
 2. Navigate to Settings
 3. In the **Account** tab, scroll down to **Personal access tokens**
 4. Enter the name of your access token in the entry form and press <kbd>Enter</kbd>
 5. Your personal access token will be created.
    * Copy this token and keep it somewhere safe.
    * **You will not get another chance to copy this token.**
-6. Paste your personal access token into the **Token ID** form in Tkinter Designer
-___
-<br>
 
 <a id="using-2"></a>
-## 2. File URL
+
+### 2. File URL
 1. In your Figma design file, click the **Share** button in the top bar, then click on **&#x1f517; Copy link**
-2. Paste the link into the **File URL** form in Tkinter Designer
-___
+
+## Using the CLI
+
+Using the CLI is as simple as installing the package and running the CLI tool.
+
+```bash
+# Example data
+$ python -m tkdesigner.cli https://www.figma.com/file/zGAo3tgCahV0n7NtqysgwY/SampleShack-TKDesigner?node-id=0%3A1 205235-fb7f1ea2-f48a-4bb1-8636-XXXXXXX -f
+
+# To learn more about how to use the cli, pass the --help flag
+$ python -m tkdesigner.cli --help
+```
+
+By default, the GUI code will be written to build/gui.py.
+
+To run the generated GUI, cd into the directory you built it to (e.g. build/) and run it just as you would any Tkinter GUI.
+```bash
+$ cd build
+$ python3 gui.py
+```
+
+## Using the GUI
+### Open Tkinter Designer before doing the following steps.
 <br>
 
-<a id="using-3"></a>
-## 3. Test Your Generated Code
-1. In Tkinter Designer, click the **Output Path** form to open a file browser
-2. Choose an output path and click **Select Folder**
-3. Press **Generate**
+1. Open TKinter Designer
```

Comment:
The previous instructions to open Tkinter Designer GUI would not be valid now. We need to update it.

Reply:
Do yourself a favor and update the link to the instructions doc as well in tkinter_designer.py . It's invalid currently.

Reply:
Yes, I marked it in my to-do for today 😄  Discovered that in the morning.

Reply:
https://github.com/ParthJadhav/Tkinter-Designer/pull/73 Done !